CHANGES.rst

```diff
@@ -1,6 +1,18 @@
 Changelog
 =========
 
+Version 2.3.4
```

Comment:
Please rebase, we already have a stub for this version in the latest 2.3-maintenance branch.